src/FsAutoComplete.Core/FileSystem.fs

```diff
@@ -140,18 +140,19 @@ type NamedText(fileName: string<LocalPath>, str: string) =
     else
       // multiline, use a builder
       let builder = new System.Text.StringBuilder()
-      // slice of the first line
+      // potential slice of the first line, including newline
+      // because we know there are lines after the first line
       let firstLine = (x :> ISourceText).GetLineString(m.StartLine - 1)
-
-      builder.Append(firstLine.Substring(m.StartColumn))
+      builder.AppendLine(firstLine.Substring(m.StartColumn))
```

Comment:
this is the bugfix - in a multiline scenario we need to append the entire first line, including newline.